### PR TITLE
Remove IValueWriter

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by simonz-bq — COMMENTED** [review]
LGTM, but I think @Armanbqt should take a look at this one.

**Comment by byronlin13** [comment]
LGTM

File: IonDotnet/IIonWriter.cs [hunk 1]
```diff
@@ -96,5 +96,106 @@ public interface IIonWriter : IValueWriter, IDisposable
         /// </summary>
         /// <param name="annotations">Set of annotations.</param>
         void SetTypeAnnotations(IEnumerable<string> annotations);
+
```
Comment (R-maan) [inline IonDotnet/IIonWriter.cs]: Not a big issue, but there is another write method ``void WriteValues(IIonReader reader);``
Maybe we can reorg the methods to have all the Write... methods together

**Review by R-maan — APPROVED** [review]
👍

File: IonDotnet/IIonWriter.cs (hunk 1 above)
Comment (ghost) [reply IonDotnet/IIonWriter.cs]: Done. I moved the `SetTypeAnnotations` to the bottom of the file, with all the other methods relating to annotations.

**Pull request closed without merging**